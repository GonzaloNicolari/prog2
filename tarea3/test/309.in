# prueba es_camino
es_camino (-1, arbol) .
insertar_en_binario (-1, arbol)
es_camino (-1, arbol) .
insertar_en_binario (1, mesa)
es_camino (-1, arbol) (2, mesa) .
insertar_en_binario (2, silla)
es_camino (-1, arbol) (2, silla) .
es_camino (-1, arbol) (2, mesa) .
es_camino (1, mesa) (2, silla) .
es_camino (-1, arbol) (1, mesa) (2, silla) .
insertar_en_binario (-20, casa)
insertar_en_binario (100, ancla)
es_camino (-1, arbol) (1, ancla) .
es_camino (-1, arbol) (100, ancla) .
es_camino (-1, arbol) (100, ancla) (-20, casa) .
es_camino (-1, arbol) (1, mesa) (-20, casa) .
Fin
