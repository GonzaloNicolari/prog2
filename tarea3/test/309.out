1>#  prueba es_camino.
2>NO es camino.
3>Insertado.
4>Es camino.
5>Insertado.
6>Es camino.
7>Insertado.
8>NO es camino.
9>NO es camino.
10>NO es camino.
11>Es camino.
12>Insertado.
13>Insertado.
14>Es camino.
15>Es camino.
16>NO es camino.
17>Es camino.
18>Fin.
